frontend/src/css/GridLogin

```diff
@@ -0,0 +1,15 @@
+.loginContainer {
```

Comment:
This file does not have an ending. And it is a mixture of html and css.

Reply:
deleted in next version